girder-dandi-archive/girder_dandi_archive/rest.py

```diff
@@ -98,3 +101,84 @@ def list_dandisets(self, limit, offset, sort):
         return Folder().find(
             {"parentId": drafts["_id"]}, limit=limit, offset=offset, sort=sort
         )
+
+    @access.public
+    @describeRoute(Description("Global Dandiset Statistics"))
+    def stats(self, params):
+        drafts = get_or_create_drafts_collection()
+        draft_count = Collection().countFolders(drafts)
+
+        # TODO no way to publish
+        published_count = 0
```

Comment:
Can you file an issue for this?

Reply:
https://github.com/dandi/dandiarchive/issues/212
That's my guess for how it would be implemented, but the whole publishing topic should probably be a much larger discussion.